pkg/db/task.go

```diff
@@ -0,0 +1,129 @@
+package db
+
+import (
+	"fmt"
+)
+
+type Task struct {
+	ID      int    `json:"id,string"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment,omitempty"`
+	Repeat  string `json:"repeat,omitempty"`
+}
+
+// DeleteTask удаляет задачу по ID
+func DeleteTask(id int) error {
+	query := `DELETE FROM scheduler WHERE id = ?`
+	result, err := DB.Exec(query, id)
+	if err != nil {
+		return fmt.Errorf("ошибка удаления задачи: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("ошибка удаления задачи: %w", err)
```

Reply:
Нужно везде поправить